cocos/2d/components/label-shadow.ts

```diff
@@ -94,7 +94,7 @@ export class LabelShadow extends Component {
      * ```
      */
     @tooltip('i18n:labelShadow.offset')
-    get offset () {
+    get offset (): Vec2 {
```

Comment:
Add readonly?

Reply:
This fix results from automatic type inference, in this PR I prefer to keep the original type unchanged, otherwise this will change the original behavior